clients/src/main/java/org/apache/kafka/clients/InFlightRequests.java

```diff
@@ -44,22 +42,18 @@ public InFlightRequests(int maxInFlightRequestsPerConnection) {
      */
     public void add(NetworkClient.InFlightRequest request) {
         String destination = request.destination;
-        Deque<NetworkClient.InFlightRequest> reqs = this.requests.get(destination);
-        if (reqs == null) {
-            reqs = new ArrayDeque<>();
-            this.requests.put(destination, reqs);
-        }
-        reqs.addFirst(request);
-        inFlightRequestCount.incrementAndGet();
+        this.requests.computeIfAbsent(destination, d -> new ArrayDeque<>()).addFirst(request);
+        inFlightRequestCount++;
     }
 
     /**
      * Get the request queue for the given node
      */
     private Deque<NetworkClient.InFlightRequest> requestQueue(String node) {
         Deque<NetworkClient.InFlightRequest> reqs = requests.get(node);
-        if (reqs == null || reqs.isEmpty())
+        if (reqs == null || reqs.isEmpty()) {
             throw new IllegalStateException("There are no in-flight requests for node " + node);
+        }
```

Comment:
Why are we making these unrelated formatting changes?

Reply:
Well, since I'm touching the file anyway, figured we could do these trivial changes.  Different projects have different tolerances for these types of change. I can revert if you'd like.

Reply:
Kafka doesn't mandate braces.

Reply:
Let's remove the brace changes please.